module-4/backend/week_one.md

```diff
@@ -1,21 +1,64 @@
 ## Week One - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week.
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. What's the most useful thing you learned from completing the intermission week work?
+
+Javascript isn't that scary! Really, though, it's still just programming.
+
 2. What are some tools to help debug JavaScript code?
+
+the debugger, using node for spiking things, the console for spiking things.
+
 3. What are some tools you need in order to unit test your JavaScript?
-4. What is the syntax for invoking a function?
-5. What is CORS?
-6. How do we enable CORS?
-7. What's `this` in JavaScript?
-8. What is Webpack and why is it useful?
-9. When/why do you want to use event delegation?
-10. What's `npm` and what do we use it for?
-
-#### Review  
-11. What's the MVC design pattern? Describe each part of MVC.
-12. What are a few ways to optimize a Rails application?
-13. What's a background worker? When would we want to use a background worker?
+
+mocha, chai, and a growth mindset.
+
+4. What is the syntax for invoking a function? Give an example.
+
+functionName()
+
+5. What's `this` in JavaScript?
+
+References the context of a given Javascript scope.
+
+6. What is Webpack and why is it useful?
+
+Webpack bundles and compiles all front end assets for a web application
+
+7. When/why do you want to use event delegation?
+
+It may allow for using a single event listener for multiple elements that may exist.
```

Comment:
I think you understand the concept here, but could be more descriptive in your explanation